projects/migrations/0007_auto_20150830_2025.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I wonder why do you have 3 migrations files? Is it because you did it in 3 steps or it created 3 in one makemigrations?
If it is the first case you could undo the migrations, remove the files and then do makemigrations again. Not necessary though